devops/ssi-helm/templates/deployment.yaml

```diff
@@ -45,6 +45,11 @@ spec:
           #   httpGet:
           #     path: /
           #     port: http
+          env:
+            {{- range $key, $value := .Values.appValues }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
```

Comment:
Is this a common pattern for helm charts?

What does the `quote` value mean?

Reply:
I pulled this pattern from what we did in rebeam. The quotes are a safety check for referencing strings in helm charts I believe: https://helm.sh/docs/howto/charts_tips_and_tricks/#quote-strings-dont-quote-integers

Reply:
https://github.com/energywebfoundation/rebeam-helm/blob/f826c617961598d4cafd57af58d497b711ed071f/templates/deployment.yaml#L50

Reply:
> I pulled this pattern from what we did in rebeam. The quotes are a safety check for referencing strings in helm charts I believe: https://helm.sh/docs/howto/charts_tips_and_tricks/#quote-strings-dont-quote-integers

Ah, I see, it's a pipe into a `quote` function. Looks good to me, thanks for the explanation!